### PR TITLE
Create mobile repo

### PR DESCRIPTION
**Pull request by maroun7**
Add .github folder
--Create workflows folder
--Create linters.yml file
--Add linters-config / html-css

Add a html file named index.html
--Created header and headline
--Create section tag
--Add menu and contact SVG icons
--Fix Webhint error
--Update html code to meet requirements

Add a css file named style.css
--Link css to html file
--Add design to header and headline using flexbox
--Add background image for section
--Apply design for SVG icons
--Fix Stylelint error
--Modify css to meet requirements of changes requested

Add a README.md file
--Update file to meet my descriptions

Add MIT.md file
--Update file and commit changes

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  background: #fff;
+  overflow: hidden;
```
Comment (henatan99) [inline style.css]: - [x] Kindly remove the height and overflow properties from the body. It hides the LET'S CONNECT title and the social links.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi Maroun, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


- [x] Please add descriptive PR summary. You left it empty. It is one of the primary things you should do to make your repo professional.
- [x] Kindly use the recommended readme template. Kindly read this reference for a professional repo. [Link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (maroun7) [reply style.css]: Thanks for the advise

**Review by henatan99 — APPROVED** [review]
Hi Maroun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**